repl/build.sbt

```diff
@@ -9,4 +9,4 @@ libraryDependencies ++= Seq(
   "org.scalatest" %% "scalatest"  % "3.0.4" % Test
 )
 
-mainClass in Compile := Some("io.idml.repl.Main")
+mainClass in Compile := Some("io.idmlrepl.Main")
```

Comment:
Is this intended? Doesn't look like it.

Reply:
it is, this got refactored a while ago into that package, but the sbt never got updated, it might move back too